server/mscalendar/user.go

```diff
@@ -168,7 +169,18 @@ func (m *mscalendar) GetRemoteUser(mattermostUserID string) (*remote.User, error
 }
 
 func (m *mscalendar) IsAuthorizedAdmin(mattermostUserID string) (bool, error) {
-	return m.Dependencies.IsAuthorizedAdmin(mattermostUserID)
+	for _, userID := range strings.Split(m.AdminUserIDs, ",") {
```

Comment:
This is a pretty unexpected change, especially for the scope of the PR. Is there value lost by removing the injected dependency here? What packages had access to this functionality that no longer have access?

Reply:
This was implemented on plugin.go, and passed to MSCalendar plugin, and only used from there. Therefore, implementing it here made more sense, and simplified the dependencies.